install.sh

```diff
@@ -0,0 +1,98 @@
+#/bin/bash
+# The script requires root permissions
+
+function install_trento() {
+    . /etc/os-release
+    if [[ ! $NAME =~ "SUSE" ]]; then
+        echo "Not SUSE operating system."
```

Comment:
I'd write here "Unsupported OS" here, no need to mention SUSE. Also, the check in the `if` would leave out `SLES`; is that intended?

Reply:
Yes, sles was left intentionally.  The installer only takes care that the trento is installed. There was a discussion when implementing an OS check in the trento itself that we cannot fully rely on the NAME field in the os-release, and we also don't need it here. If the version is not supported, the user will see it in the Web Cockpit.

Reply:
Changed to "This is an trento agent installer"